blazor/treeview/integration.md

```diff
@@ -0,0 +1,1133 @@
+                Name = "Sanjay",
+            });
+
+        ListData.Add(new EmployeeData
+            {
+                Id = "4",
+                Pid = "1",
+                Name = "Steve",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "6",
+                Pid = "1",
+                Name = "Martin",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "7",
+                Name = "Laura",
+                HasChild = true,
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "8",
+                Pid = "7",
+                Name = "Mic",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "9",
+                Pid = "7",
+                Name = "Nancy",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "10",
+                Pid = "7",
+                Name = "Andrew",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "11",
+                Name = "Robert King",
+                HasChild = true,
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "12",
+                Pid = "11",
+                Name = "Richard",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "13",
+                Pid = "11",
+                Name = "James",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "14",
+                Pid = "11",
+                Name = "Murrey",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "15",
+                Pid = "11",
+                Name = "Chris",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "16",
+                Name = "Margaret Peacock",
+                HasChild = true,
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "17",
+                Pid = "16",
+                Name = "Ryaz",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "18",
+                Pid = "16",
+                Name = "Mary",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "19",
+                Pid = "16",
+                Name = "Stephen",
+            });
+        ListData.Add(new EmployeeData
+            {
+                Id = "20",
+                Pid = "16",
+                Name = "Raffel",
+            });
+    }
+}
+
+```
+
+## Badge
+
+The Blazor TreeView component can integrate with the Badge component and allows for customizing the node structure using the NodeTemplate tag, which allows for creating a tree node with any custom structure, such as a web mail folder with a name and unread message count using the Badge component.

```

Comment:
Link for NodeTemplate?

Reply:
Don't have the link for the Blazor NodeTemplate property.